11.coreboot.u-root.systemboot/README.md

```diff
@@ -43,7 +43,39 @@ For example, you may want to include static builds of `kexec` or `flashrom`, tha
 
 ## Building a suitable Linux kernel
 
-TODO
+TODO add sample `.config`.
+
+You need a relatively recent kernel. Ideally a kernel 4.16, to have support for VPD variables, but a 4.11 can do the job too, if you don't care about boot entries and want "brute-force" booting only.
+
+We will build a kernel with the following properties:
+
+* small enough to fit most flash chips
+* that can run Go programs (mainly futex and epoll support)
```

Comment:
I think you'll have to get more specific here. In fact I hate to say it but the things Go needs depend on which version of Go you are using. 

What would be nice is a very minimal six lines or so .config inline right here that you copy to your linux .config, then do make menuconfig, then it's all ready. This is known to work (I've done it).

Reply:
The `.config` is mentioned in the TODO on top, I plan to attach an actual file (or inline it if it's small enough). Would that be OK?

Reply:
yeah, just be aware, you only need about 6-8 lines of config, then you let the kernel fill in the rest. This is also useful as it makes it less version dependent.